app/index.js

```diff
@@ -726,9 +776,68 @@ module.exports = generators.Base.extend({
     }
   },
 
+  generateSDKs: function() {
```

Comment:
What we've been doing for this sort of self-contained generation work is to add it in the default phase, between `initializing` and `writing`. We've been using the prefix `build` to denote this sort of activity, for an example, see `buildSwagger` in  `refresh/index.js`.

(Aside: for logic that is closely coupled to the writing of files, we've been including it within an action inside the `writing` phase (usually prefixed with `create` or `write`). For an example, see `createCRUD` in `refresh/index.js`.)